tests/SprykerEcoTest/Zed/CrefoPay/Business/StartCrefoPayTransactionFacadeTest.php

```diff
@@ -42,5 +57,33 @@ public function doTest(QuoteTransfer $quoteTransfer): void
         $this->assertNotEmpty($crefoPayTransactionTransfer->getCrefoPayOrderId());
         $this->assertNotEmpty($crefoPayTransactionTransfer->getSalt());
         $this->assertGreaterThan(0, count($crefoPayTransactionTransfer->getAllowedPaymentMethods()));
+
+        if ($useIndependentOrderId) {
```

Comment:
If/else is not really convenient construction to use in the test. Please, consider splitting the test in two.